Source/Fluids/WarpXFluidContainer.H

```diff
@@ -0,0 +1,172 @@
+/* Copyright 2023 Grant Johnson, Remi Lehe
+ *
+ * This file is part of WarpX.
+ *
+ * License: BSD-3-Clause-LBNL
+ */
+#ifndef WARPX_WarpXFluidContainer_H_
+#define WARPX_WarpXFluidContainer_H_
+
+#include "Evolve/WarpXDtType.H"
+#include "Initialization/PlasmaInjector.H"
+#include "MultiFluidContainer.H"
+
+#include<AMReX_MultiFab.H>
+#include<AMReX_Vector.H>
+
+#include <string>
+
+
+/**
+ * WarpXFluidContainer is the base polymorphic class from which all concrete
```

Comment:
This docstring was probably copied from WarpXParticleContainer ; I don't think that it is correct: here `WarpXFluidContainer` is not a polymorphic class. This would need to be edited.

Reply:
Removed "polymorphic" and updated the description line below.